### ISSUE TITLE
libtbb.so causing build failures on some systems

### ISSUE THREAD

**Issue by villaa** [villano-lab/nrCascadeSim#93]
On my Linux Ubuntu 18.04 system there is a build error even though the Travis builds (which also use Ubuntu 18.04) don't have  it. 

The error looks like:

```
[ 69%] Building CXX object CMakeFiles/libncap.dir/src/cascadeProd.cpp.o
make[2]: *** No rule to make target '/mnt/build/ws/BUILDTYPE/Release/LABEL/ROOT-ubuntu18.04/build/lib/libtbb.so', needed by 'lib/libncap.so'.  Stop.
CMakeFiles/Makefile2:407: recipe for target 'CMakeFiles/libncap.dir/all' failed
make[1]: *** [CMakeFiles/libncap.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

On the `make` step.

**Comment by villaa** [comment]
Here are some links that might be associated with this problem, surprise, it may be a `ROOT` problem.  

[link1](https://root-forum.cern.ch/t/error-linking-against-binary-disrtibution-of-root/33648)

[link2](https://sft.its.cern.ch/jira/browse/ROOT-10030)

**Comment by villaa** [comment]
According to the first post the _binary_ distributions of `ROOT` contain `libtbb.so` in their library directory. This might explain why it is passing on Travis.

**Comment by villaa** [comment]
Don't quite know why but this issue seemed to evaporate after PR #95

**Comment by villaa** [comment]
Addressed by PR #95 -- a bit mysterious though.
